src/bp/core/database/migrations/1548770808648_srv_channel_users.ts

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  up: function (knex) {
+    return knex.schema.alterTable('srv_channel_users', function (table) {
+      table.string('botId')
+      table.integer('timezone')
+    })
+  },
+
+  down: function (knex) {
+    return knex.schema.alterTable('srv_channel_users', function (table) {
+      table.dropColumn('botId')
+      table.dropColumn('timezone')
+    })
+  }
+}
```

Comment:
@dmytropanontko , can this migration go with `broadcast` module improvements? It's not relevant to current task.